packages/sidecar/package.json

```diff
@@ -52,6 +52,6 @@
     "extends": "../../package.json"
   },
   "engines": {
-    "node": ">=18"
+    "node": ">=20"
```

Comment:
are we dropping support for node 18? This would be a breaking change because Node 18 isn't EOL yet. afaik setting `engines` will only produce a warning in conventional `npm install` runs but if we actually use node 20+ only features, we might still run into incompatibilities.

Reply:
I'm happy to let you make the call on that! Just wanted to mention that users who are currently using spotlight in a node 18 project (for example, by calling `Spotlight.init()`) might get warnings about spotlight being incompatible if they update their spotlight packages, due to `engines` sepcifying Node 20+. Depending on their npm/package manager settings, this could also throw an error. In SDK-land, we treat this as a breaking change.

Reply:
Oh wait, I think I can just revert this but keep the main repo Node version at 22 for many things. I'll see if I can set up a test matrix.

Reply:
All fair calls btw, I'm just treating Spotlight a bit looser than SDKs as it is quite new, not widely adopted, and the stakes are much lower compared to SDKs.

Reply:
That's also fair, no worries!